mlir/test/Dialect/OpenMP/invalid.mlir

```diff
@@ -639,90 +639,6 @@ cleanup {
 
 // -----
 
-func.func @foo(%lb : index, %ub : index, %step : index) {
```

Comment:
The errors teste here are probably still reasonable. Can you retain and remove the omp.reduction operation only?